src/Games/brain-cals.js

```diff
@@ -0,0 +1,31 @@
+import getRandomNumber from '../randomCount.js';
+import theCoreOfGames from '../index.js';
+
+const brainRulls = () => {
+  const descriptionOfGame = 'What is the result of the expression?';
+
+  const randomNumberOne = getRandomNumber(1, 10);
+  const randomNumberTwo = getRandomNumber(1, 10);
+  const operator = ['+', '-', '*'];
+  const randomOperator = getRandomNumber(0, 2);
```

Comment:
I see a random number, not an operator, plz rename to a better name

Reply:
ok, changed, now it is 'choiceRandomOperator'

Reply:
operatorIndex?

Reply:
understand, index